### ISSUE TITLE
Sea of Thieves - Value does not fall within the expected range. :(

### ISSUE THREAD

**Issue by litteringand33** [BrianLima/UWPHook#29]
Greetings!

First of all, thanks for creating this app! It really seems like it could be very useful :)

Having said that, I seem to be getting the same issue that this guy was getting 

https://github.com/BrianLima/UWPHook/issues/24

I am using the latest version, and same as the other guy, when using shell:appsFolder\Microsoft.SeaofThieves_8wekyb3d8bbwe!AthenaClientShipping as a launch condition for explorer.exe does work

Uploaded screenshots in case they help :)

https://imgur.com/a/Sk9Pp0Y

Thanks again!

**Comment by SubSonix** [comment]
+1

**Comment by litteringand33** [comment]
Hmmm ok I can't seem to update from 2.3.0 to 2.5.1, there must be something I'm doing wrong.... :(

I download the files from https://github.com/BrianLima/UWPHook/releases/tag/2.5.1

If I just run UWPHook.exe it installs UWPhook onto my PC, but it's version 2.3.0 when I check the settings...

When it comes to performing the update, I'm honestly not quite sure what I'm supposed to do :(

I've tried copying the UWPHook folder from the UWPHook-2.5.1.zip to my install directory in Program Files, but it still shows as version 2.3.0 in UWPHook.

I've tried running the setup.msi located in UWPHook-2.5.1.zip, and UWPHook still shows as 2.3.0 in the program settings (and Sea of Thieves gives the same error....)

Sorry if I'm being a complete noob/moron here, but I would most definitely appreciate a nudge in the right direction if possible :)

Thanks again!

**Comment by SubSonix** [comment]
I think that might just be something that was overlooked by the dev? Perhaps.

**Comment by BrianLima** [comment]
Hi guys. really sorry for that, i`m looking into it now.

**Comment by BrianLima** [comment]
Yep, i screwed it up, just uploaded the correct binary, while the version here on github is 2.5.1.1, it will 
correctly show 2.5.1.0 at the settings window now.
https://github.com/BrianLima/UWPHook/releases/tag/2.5.1.1

![image](https://user-images.githubusercontent.com/1337383/44372737-af331400-a4bb-11e8-8767-02a823a99c65.png)

Sorry for the inconvenience guys.

**Comment by SubSonix** [comment]
Thank you for the update :)

**Comment by mikicof** [comment]
Sorry, but I'm on version 2.7 and I'm having the same problem, tried installing 2.5.1.1 or 2.6 but the same problem.

Also, I've added manually the game (adding UWPHook and changing launch params) because when I load the list it shows a "Long path" error and doesn't load all uwp apps...
Anyway, I tried with the loaded ones like Rage 2 and works fine.

Thank you!
